nopol/src/main/java/fr/inria/lille/spirals/evo/processors/TestSelectionProcessor.java

```diff
@@ -0,0 +1,59 @@
+package fr.inria.lille.spirals.evo.processors;
+
+import java.lang.annotation.Annotation;
+import java.util.List;
+
+import spoon.processing.AbstractProcessor;
+import spoon.reflect.declaration.CtAnnotation;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.declaration.CtElement;
+import spoon.reflect.declaration.CtMethod;
+
+/**
+ * This processor remove all method which are not contains in keptMethod and methodTested
```

Comment:
This processor removes all methods that are not  methodTested and not in keptMethods list.